soccer/ui/MainWindow.ui

```diff
@@ -82,6 +82,9 @@
              <height>24</height>
```

Comment:
The ui jiggles for me when the numbers change, they probably need padding around them or something. I can work on that when I get some free time...